.github/workflows/test.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       - name: Setting up Node.js ${{ matrix.node-version }}
```

Comment:
Side note.
There is no matrix context here (same for line 35) :smile:

Reply:
Alright, lets clean it up a bit more 👍

Reply:
Removed the matrix context vars, additionally removed the caching stuff (as that is provided by the setup-node action nowadays).